library/Vanilla/Web/Asset/LegacyAssetModel.php

```diff
@@ -21,33 +21,25 @@
  * Manages Assets.
  */
 class LegacyAssetModel extends Gdn_Model {
-    /**
-     * The number of seconds to wait after a deploy before switching the cache buster.
-     */
-    const CACHE_GRACE_PERIOD = 90;
-
-    /** @var string Directory for webpack-built files. */
-    const WEBPACK_DIST_DIRECTORY_NAME = "dist";
-
-    /** @var string Webpack built script extension. */
-    const WEBPACK_SCRIPT_EXTENSION = ".min.js";
-
     /** @var array List of CSS files to serve. */
     protected $_CssFiles = [];
 
      /** @var string */
     public $UrlPrefix = '';
 
+    /** @var string */
+    private $cacheBuster;
+
     /**
-     * @var \Vanilla\AddonManager
+     * LegacyAssetModel constructor.
+     *
+     * @param string $cacheBuster
      */
-    private $addonManager;
-
-    public function __construct(\Vanilla\AddonManager $addonManager) {
+    public function __construct(string $cacheBuster) {
```

Comment:
The AddonManager even being added to it was a pretty recent development (2.7) and should only have ever been instantiated through the container. Most usage of the this class was it's static methods.

Theoretically someone could have been using this in open source if they were working against master, but we never publicly documented any of the public methods. I don't have particular concern as a result.

Reply:
`AddonManager` has been injected here since 05519ce8f2617a70ca19841ce411f39da1d613e9 (v2.6), but it sounds like it didn't see any real usage in Vanilla until 2.7. That, coupled with you not perceiving any real risk here, I'm gonna call this "resolved".